test/e2e/components/vl-button.js

```diff
@@ -4,11 +4,11 @@ const { By } = require('selenium-webdriver');
 class VlButtonElement extends VlElement {  
 
     async getIcon() {
-        const icon = await this.findElement(By.css(this.selector + ' [is="vl-icon"]'));
-        if (icon)  {
+        const icon = await this.findElement(By.css('span[is="vl-icon"]'));
```

Comment:
Je kan die span zelfs weglaten denk ik?

```suggestion
        const icon = await this.findElement(By.css('[is="vl-icon"]'));
```

Reply:
Ja mogelijk beter, misschien zijn er (in de toekomst?) ook vl-icons met iets anders dan span.